rest-api-spec/src/main/resources/rest-api-spec/api/get_all_pits.json

```diff
@@ -0,0 +1,19 @@
+{
+  "get_all_pits":{
+    "documentation":{
+      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/point_in_time/",
+      "description":"Get all active point in time searches."
+    },
+    "stability":"experimental",
```

Comment:
Why are we marking the API as experimental? Is the feature experimental?

Reply:
Since we don't support older versions we'll mark this as experimental without which bwc tests will fail